lib/hider/crypt/argon2.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Hider.Crypt.Argon2 do
+  @salt :hider |> Application.compile_env!([:argon2, :salt]) |> Base.decode64!()
```

Comment:
Use the same salt for all, so it's kind of a secret key that only our database should know.